src/components/RestaurantList/RestaurantList.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <v-container class="card-container d-flex flex-wrap justify-center">
+    <v-row>
+      <v-col
+        cols="12"
+        md="4"
+        v-for="restaurant in restaurants"
+        v-bind:key="restaurant.id"
+      >
+        <restaurant-card v-bind:restaurant="restaurant" />
+      </v-col>
+    </v-row>
+  </v-container>
+</template>
```

Comment:
Looks good on very big screens (PC) and very small screens (typical smartphones).

For small-ish vertical screens, e.g. big smartphones or ebook readers, the cards still take the whole screen, like on small screens, but it makes the images huge and therefore almost all of the screen is took by the image.

For big-ish vertical screens, as well as for small horizontal screens, e.g. small to medium tablets, the cards are very small, to such an extent that the rating and number of votes end up as two separate lines of text.